FirebaseCoreExtension.podspec

```diff
@@ -34,5 +34,5 @@ Pod::Spec.new do |s|
       "#{s.module_name}_Privacy" => 'FirebaseCore/Extension/Resources/PrivacyInfo.xcprivacy'
     }
 
-    s.dependency 'FirebaseCore', '~> 11.0'
+    s.dependency 'FirebaseCore', '>= 11.4.1', '< 12.0'
```

Comment:
@andrewheard , going to revert this because it will make 11.4.1 incompatible with Firebase Core 11.2, which then require a Flutter update.